urbanairship-react-native/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "jsx": "react-native",
+    "lib": ["es2015"],
+    "moduleResolution": "node",
+    "strict": true,
+    "target": "es5",
+    "noImplicitAny": true
```

Comment:
This is an important setting that's false by default. Otherwise variables whose type can't be inferred will be treated as `any`, which is convenient for translating vanilla JS to TS, but covers up issues code that can (and should) be made more type-safe.